src/min_fire.cpp

```diff
@@ -250,7 +250,15 @@ int MinFire::iterate(int maxiter)
     // sync across replicas if running multi-replica minimization
 
     if (update->ftol > 0.0) {
-      fdotf = fnorm_sqr();
+      if (normstyle == 1) {		// max force norm
```

Comment:
Better to use a enum value for normstyle == 1, e.g. normstyle == INFINITY or normstyle == EUCLIDEAN

Reply:
In case we add other options later.